tests/devices/test_default_qubit.py

```diff
@@ -1483,13 +1489,14 @@ def test_hermitian(self, theta, phi, varphi, tol):
 
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
+
 @pytest.mark.parametrize("theta, phi, varphi", list(zip(THETA, PHI, VARPHI)))
 class TestTensorSample:
```

Comment:
This was a bug uncovered: With the previous logic the result was deterministic, even though sampling was tested!